vMenuServer/MainServer.cs

```diff
@@ -894,5 +895,31 @@ private void AddTeleportLocation([FromSource]Player source, string locationJson)
         }
         #endregion
 
+        #region Dump language text
+        /// <summary>
+        /// Dump the language text
+        /// </summary>
+        /// <param name="source"></param>
+        /// <param name="text"></param>
+        private void DumpLanguages([FromSource] Player source, string text)
+        {
+            if (IsPlayerAceAllowed(source.Handle, "vMenu.DumpLanguages.Dump") || IsPlayerAceAllowed(source.Handle, "vMenu.Everything") ||
```

Comment:
This does not need to be cheater checked if you make this a server command. It's also not a destructive action so there should be no need to have this be cheater proof.